FU_NSSDB/NSSdbOps.cs

```diff
@@ -238,7 +238,7 @@ private string getMySqlSql(SQLType type)
                     results = @"INSERT INTO ""nss"".""Regions""(""Name"",""Code"") VALUES('{0}','{1}')";
                     break;
                 case SQLType.e_regressionregion:
-                    results = @"INSERT INTO ""nss"".""RegressionRegions""(""Name"",""Code"",""Description"",""CitationID"") VALUES('{0}', '{1}', '{2}',{3})";
+                    results = @"INSERT INTO ""nss"".""RegressionRegions""(""Name"",""Code"",""Description"",""CitationID"",""StatusID"") VALUES('{0}', '{1}', '{2}',{3}, 4)";
```

Comment:
is 4 suppose to be the statusID? or will this be passed in as an argument? The way it is written 4 will be the statusID for all regressionRegions by default. It'll probably be easier to maintain long term to pass the argument [here ](https://github.com/USGS-WiM/NSSServices/blob/KJ-114/FU_NSSDB/ForceUpdate.cs#L221)in as a value rather than setting the default in the sql. (just a suggestion, not necessary)

Reply:
Yeah this is correct, Pete wants all regression regions coming from SSDB to be set to the statusID of "4" (SS approved) by default, but if I'm going in update things anyway, I might move it to the location you linked.